x/leverage/keeper/borrows.go

```diff
@@ -114,8 +97,7 @@ func (k Keeper) SupplyUtilization(ctx sdk.Context, denom string) sdk.Dec {
 // CalculateBorrowLimit uses the price oracle to determine the borrow limit (in USD) provided by
 // collateral sdk.Coins, using each token's uToken exchange rate and collateral weight.
 // An error is returned if any input coins are not uTokens or if value calculation fails.
-// If the historic parameter is true, uses medians of recent prices instead of current prices.
-func (k Keeper) CalculateBorrowLimit(ctx sdk.Context, collateral sdk.Coins, historic bool) (sdk.Dec, error) {
+func (k Keeper) CalculateBorrowLimit(ctx sdk.Context, collateral sdk.Coins, mode types.PriceMode) (sdk.Dec, error) {
```

Comment:
I'd update the godoc to describe `mode types.PriceMode` here as well

Reply:
Actually managed to take the `mode` argument out of that function, since it always uses `PriceModeLow`. Now godoc explains that instead.